docker/default.env

```diff
@@ -24,3 +24,6 @@ WIS2BOX_URL=http://localhost:8999
 # mappings of topic hierarchy to wis2box data plugins
 # optionally override default mappings from wis2box data plugins
 # WIS2BOX_DATADIR_DATA_MAPPINGS=${PWD}/wis2box-data-mappings.yml
+
+# access control
+WIS2BOX_AUTH_STORE=/data/wis2box/auth.db
```

Comment:
Add this env var to the docs (https://wis2box.readthedocs.io/en/latest/configuration.html#other)